cibyl/outputs/cli/ci/env/factory.py

```diff
@@ -24,7 +24,7 @@ class CIPrinterFactory:
     """
 
     @staticmethod
-    def from_style(style, query, verbosity):
+    def from_style(style, query, verbosity, complete=False):
```

Comment:
I would not give 'complete' a default here. Let however calls this have to think about it. That is up to you though, how you like it better.

Reply:
You're right. I removed the default value here